packages/wdio-logger/src/node.ts

```diff
@@ -26,37 +26,37 @@ const SERIALIZERS = [{
     /**
      * display error stack
      */
-    matches: (err) => err instanceof Error,
-    serialize: (err) => err.stack
+    matches: (err: any) => err instanceof Error,
+    serialize: (err: any) => err.stack
 }, {
     /**
      * color commands blue
      */
-    matches: (log) => log === matches.COMMAND,
-    serialize: (log) => chalk.magenta(log)
+    matches: (log: string) => log === matches.COMMAND,
+    serialize: (log: string) => chalk.magenta(log)
 }, {
     /**
      * color data yellow
      */
-    matches: (log) => log === matches.DATA,
-    serialize: (log) => chalk.yellow(log)
+    matches: (log: string) => log === matches.DATA,
+    serialize: (log: string) => chalk.yellow(log)
 }, {
     /**
      * color result cyan
      */
-    matches: (log) => log === matches.RESULT,
-    serialize: (log) => chalk.cyan(log)
+    matches: (log: string) => log === matches.RESULT,
+    serialize: (log: string) => chalk.cyan(log)
 }]
 
 const loggers = log.getLoggers()
-let logLevelsConfig = {}
+let logLevelsConfig: Record<string, log.LogLevelDesc> = {}
 const logCache = new Set()
-let logFile
+let logFile: fs.WriteStream
 
 const originalFactory = log.methodFactory
-const wdioLoggerMethodFactory = function (methodName, logLevel, loggerName) {
-    const rawMethod = originalFactory(methodName, logLevel, loggerName)
-    return (...args) => {
+const wdioLoggerMethodFactory = function (this: log.Logger, methodName: string, logLevel: log.LogLevelDesc, loggerName: string) {
+    const rawMethod = originalFactory(methodName, logLevel as log.LogLevelNumbers, loggerName)
```

Comment:
I see you casting the logLevel values a lot, I feel like we might need to expand the type of what the logLevel can be.